docs/how-to.md

```diff
@@ -79,10 +79,29 @@ This library doesn't implement all RPC methods, so if you're in dire need of usi
 :end-before: end example
 ```
 
+## Use with _R_ via [reticulate]
+
+If you prefer using _R_ instead of _Python_, you can use the `reticulate` package to get survey data downloaded with `citric`.
```

Comment:
```suggestion
If you prefer using _R_ instead of _Python_ to analyze your survey data, you can use the `reticulate` package to get survey data downloaded with `citric`.
```